src/module/webauthn-validator/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { hexToBytes, bytesToHex, type Hex } from 'viem'
+import { PublicKey, WebauthnSignature } from './types'
+
+export function parsePublicKey(publicKey: Hex | Uint8Array): PublicKey {
```

Comment:
do you think end users will ever need to use these? if yeah then lets export them all the way too but my hunch is they wont

Reply:
Don't think they. In the event they do, they can lean on [`parseKey`](https://github.com/wevm/webauthn-p256?tab=readme-ov-file#parsepublickey) / [`parseSignature`](https://github.com/wevm/webauthn-p256?tab=readme-ov-file#parsepublickey) from the `webauthn-P256` package. But this is deliberately designed to abstract this step parsing step into an implementation detail 🤷‍♂️